src/main/java/frc/robot/Controls.java

```diff
@@ -448,30 +451,55 @@ private Command getAlgaeIntakeCommand() {
     };
   }
 
-  private Command getCoralBranchHeightCommand() {
-    return switch (branchHeight) {
-      case CORAL_LEVEL_FOUR -> superStructure.coralLevelFour(driverController.rightBumper());
-      case CORAL_LEVEL_THREE -> superStructure.coralLevelThree(driverController.rightBumper());
-      case CORAL_LEVEL_TWO -> superStructure.coralLevelTwo(driverController.rightBumper());
-      case CORAL_LEVEL_ONE -> superStructure.coralLevelOne(driverController.rightBumper());
-    };
-  }
-
-  private Command getSoloCoralBranchHeightCommand() {
-    return switch (branchHeight) {
-      case CORAL_LEVEL_FOUR -> superStructure
-          .coralLevelFour(soloController.rightBumper())
-          .andThen(() -> soloScoringMode = soloScoringMode.NO_GAME_PIECE);
-      case CORAL_LEVEL_THREE -> superStructure
-          .coralLevelThree(soloController.rightBumper())
-          .andThen(() -> soloScoringMode = soloScoringMode.NO_GAME_PIECE);
-      case CORAL_LEVEL_TWO -> superStructure
-          .coralLevelTwo(soloController.rightBumper())
-          .andThen(() -> soloScoringMode = soloScoringMode.NO_GAME_PIECE);
-      case CORAL_LEVEL_ONE -> superStructure
-          .coralLevelOne(soloController.rightBumper())
-          .andThen(() -> soloScoringMode = soloScoringMode.NO_GAME_PIECE);
-    };
+  private Command getCoralBranchHeightCommand(String version) {
```

Comment:
Ideally, this should be an enum instead of a string.  With a string you can get typos in the code and the compiler will happily accept it.  With enums, it would throw on compile on a typo.